ci-operator/config/rh-ecosystem-edge/nvidia-ci/rh-ecosystem-edge-nvidia-ci-main.yaml

```diff
@@ -10,7 +10,7 @@ build_root:
     tag: golang-1.17
 images:
 - dockerfile_path: Containerfile
-  to: nvidia-ci-tools
+  to: nvidia-ci
```

Comment:
@ybettan do we want also to include `promotion`  to publish the image?

```
promotion:
  to:
  - name: latest
    namespace: nvidia-ci
```

We can also mirror it it to quay.io. WDYT?